cypress/integration/examples/ErrorBoundary.spec.js

```diff
@@ -0,0 +1,26 @@
+/// <reference types="cypress" />
+
+context('ErrorBoundary', () => {
```

Comment:
I feel like we should name this `suspense` tests since that's the main thing we're testing here. Thoughts?

Reply:
you're is right.